tests/api.c

```diff
@@ -42188,6 +42256,163 @@ static void test_wolfSSL_CTX_get_min_proto_version(void)
 #endif /* defined(OPENSSL_EXTRA) || defined(OPENSSL_ALL) */
 }
 
+static void test_wolfSSL_security_level()
+{
+#if defined(OPENSSL_EXTRA)
+    printf(testingFmt, "test_wolfSSL_security_level()");
+    
+    SSL_CTX *ctx;
+    
+    #ifdef WOLFSSL_TLS13
+        #ifdef NO_WOLFSSL_SERVER
+            AssertNotNull(ctx = wolfSSL_CTX_new(wolfTLSv1_3_client_method()));
+        #else
+            AssertNotNull(ctx = wolfSSL_CTX_new(wolfTLSv1_3_server_method()));
+        #endif
+        SSL_CTX_set_security_level(ctx, 1);
+        AssertTrue(1);
+        
+        AssertIntEQ(SSL_CTX_get_security_level(ctx), 0);
+        
+        SSL_CTX_free(ctx);
+    #else
+        (void)ctx;
+    #endif
+    
+    printf(resultFmt, passed);
+#endif
+}
+
+static void test_wolfSSL_SSL_in_init()
+{
+#if defined(OPENSSL_ALL) && !defined(NO_BIO)
+    printf(testingFmt, "test_wolfSSL_SSL_in_init()");
+    
+    SSL_CTX* ctx;
+    SSL*     ssl;
+    const char* testCertFile;
+    const char* testKeyFile;
+    
+    #ifdef WOLFSSL_TLS13
+        #ifdef NO_WOLFSSL_SERVER
+            AssertNotNull(ctx = wolfSSL_CTX_new(wolfTLSv1_3_client_method()));
+        #else
+            AssertNotNull(ctx = wolfSSL_CTX_new(wolfTLSv1_3_server_method()));
+        #endif
+#ifndef NO_RSA
+        testCertFile = svrCertFile;
+        testKeyFile = svrKeyFile;
+#elif defined(HAVE_ECC)
+        testCertFile = eccCertFile;
+        testKeyFile = eccKeyFile;
+#endif
+        if  (testCertFile != NULL && testKeyFile != NULL) {
+            AssertTrue(SSL_CTX_use_certificate_file(ctx, testCertFile,
+                                                    SSL_FILETYPE_PEM));
+            AssertTrue(SSL_CTX_use_PrivateKey_file(ctx, testKeyFile,
+                                                    SSL_FILETYPE_PEM));
+        }
+        
+        ssl = SSL_new(ctx);
+        AssertNotNull(ssl);
+        AssertIntEQ(SSL_in_init(ssl), 1);
+        
+        SSL_CTX_free(ctx);
+        SSL_free(ssl);
+    #else
+        (void)ctx;
+        (void)ssl;
+        (void)testCertFile;
+        (void)testKeyFile;
+    #endif
+    
+    printf(resultFmt, passed);
+#endif
+}
+
+static void test_wolfSSL_EC_curve()
+{
+#if defined(OPENSSL_EXTRA) && defined(HAVE_ECC)
+    printf(testingFmt, "test_wolfSSL_EC_curve()");
+    int nid = NID_secp160k1;
+    const char* nid_name;
+    
+    AssertNotNull(nid_name = EC_curve_nid2nist(nid));
+    AssertIntEQ(XMEMCMP(nid_name, "K-160", XSTRLEN("K-160")), 0);
+    
+    AssertIntEQ(EC_curve_nist2nid(nid_name), nid);
+    
+    printf(resultFmt, passed);
+#endif
+}
+
+static void test_wolfSSL_OpenSSL_version()
+{
+#if defined(OPENSSL_EXTRA)
+    printf(testingFmt, "test_wolfSSL_OpenSSL_version()");
+    const char* ver;
+    
+    #if defined(WOLFSSL_QT)
```

Comment:
Should switch on OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT

Reply:
Guard by OPNESSL_VESION_NUMBER instead of WOLFSSL_QT